### PR TITLE
Home

### PR DESCRIPTION
**Pull request by saadahmadhere**
This PR consists of the landing page design.

### CODE REVIEW

**Review by AnuragSahu11 — APPROVED** [review]
Good choice of colors, good use of calc(), ready to merge.

File: src/Screens/Home/Home.css [hunk 1]
```diff
@@ -0,0 +1,56 @@
+.main_home {
+  width: 100%;
+}
+
+.main_home {
```
Comment (junaidshaikh-js) [inline src/Screens/Home/Home.css]: Element is targeted twice

**Review by junaidshaikh-js — APPROVED** [review]
Good work. 

Suggested one change if you had done it by mistake. Otherwise good to merge.

**Review by abhij1607 — COMMENTED** [review]
Liked the work done with font-size responsiveness | | Good practices followed

File: src/Screens/Home/Home.css (hunk 1 above)
Comment (saadahmadhere) [reply src/Screens/Home/Home.css]: Thanks for pointing, resolved in https://github.com/saadahmadhere/Stopphur/pull/3/commits/974e2723d8d7d3d3efab0c6f1723715ab05a0fa7

**Pull request merged**